server/datastore/mysql/app_configs.go

```diff
@@ -67,14 +67,14 @@ func (d *Datastore) SaveAppConfig(ctx context.Context, info *fleet.AppConfig) er
 		}
 
 		return nil
-	})
+	}), "")
```

Comment:
Should we make the string optional with `...string`? Ideally, we wouldn't have to write anything, the generated context would tell us everything we need to know.

Reply:
It's great to allow for messages, but it would be good to not need to, and so you can just do `return ctxerr.Wrap(ctx, err)`, which can even be templated and have a shortcut in our respective IDEs :)

Reply:
:+1: Yeah that'll be a bit nicer.